### PR TITLE
Quickstart notebooks to demonstrate training a model using JAX on MNIST with Syft

### PR DESCRIPTION
**Pull request by khoaguin**
According to [this ticket](https://github.com/orgs/OpenMined/projects/81/views/1?pane=issue&itemId=31132573).
The notebooks are in `notebooks/quickstart/model_training_using_syft_and_jax`

### CODE REVIEW

**Comment by rasswanth-s** [comment]
Wohoo ! 🙌

File: notebooks/quickstart/model_training_using_syft_and_jax/Data Owner (DO).ipynb [hunk 1]
```diff
@@ -0,0 +1,1918 @@
...
..., 0.0, 1.0], [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0], [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0], [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0], [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.
```
Comment (shubham3121) [inline notebooks/quickstart/model_training_using_syft_and_jax/Data Owner (DO).ipynb]: Can we create a link to the notebook as well

File: notebooks/quickstart/model_training_using_syft_and_jax/Data Scientist (DS).ipynb [hunk 2]
```diff
@@ -0,0 +1,1894 @@
...
+   "source": [
+    "project.requests[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "987026f3",
+   "metadata": {},
+   "source": [
+    "### 📓 Now switch back to the DO's notebook"
```
Comment (shubham3121) [inline notebooks/quickstart/model_training_using_syft_and_jax/Data Scientist (DS).ipynb]: Similarly, link to DO's notebook.

**Review by shubham3121 — CHANGES_REQUESTED** [review]
Superb @khoaguin 🔥 🎉 !!!

The notebooks look good. Can we move them to `notebooks/tutorials/model_training/`

File: notebooks/quickstart/model_training_using_syft_and_jax/Data Owner (DO).ipynb (hunk 1 above)
Comment (khoaguin) [reply notebooks/quickstart/model_training_using_syft_and_jax/Data Owner (DO).ipynb]: done @shubham3121

File: notebooks/quickstart/model_training_using_syft_and_jax/Data Scientist (DS).ipynb (hunk 2 above)
Comment (khoaguin) [reply notebooks/quickstart/model_training_using_syft_and_jax/Data Scientist (DS).ipynb]: done

**Review by shubham3121 — APPROVED** [review]

**Pull request merged**